src/cli/command-package.ts

```diff
@@ -29,8 +29,19 @@ export default async function run(options: CliOptions) {
 
   // Ok, we're updating a package in a directory not under our control.
   // We need to be sure it's safe.
-  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s']);
-  if (!options.force && (stdout || stderr)) {
+  let stdout, stderr;
+  try {
+      ({stdout, stderr} = await exec(inDir, 'git', ['status', '-s']));
+  } catch (e) {
+    // Grab command execution results from exception info.
+      ({stdout, stderr} = e);
+  }
+  const isRepo = !((stderr || '').indexOf('Not a git repository') !== -1);
+  if (!isRepo) {
```

Comment:
seems easier to reason about to move this code into the catch block above. Assume that if the command succeeded that `isRepo` is true.